src/pages/juniorPromise/components/SeniorListBackground.tsx

```diff
@@ -58,6 +90,39 @@ const BtnLayout = styled.div`
   padding: 0.7rem 2rem;
 `;
 
+const SelectedChipList = styled.div`
+  display: flex;
+  gap: 0.8rem;
+  align-items: center;
+
+  /* 필요에 따라 가로 스크롤바 생성 */
+  overflow: scroll hidden;
```

Comment:
P1) overflow: scroll hidden;는 올바른 CSS 문법이 아니라고 해요! overflow 속성은 각각 overflow-x와 overflow-y로 나누어서 가로와 세로 스크롤을 개별적으로 제어해야 한다네요~!

```suggestion
overflow-x: scroll;
overflow-y: hidden;

```

Reply:
아 그것이.. 그렇게 했는데 자동으로 병합되더라구요,,감사합니도!!!